src/main/java/karuna/karuna_backend/Config/Filters/JwtFilter.java

```diff
@@ -0,0 +1,81 @@
+package karuna.karuna_backend.Config.Filters;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.jsonwebtoken.ExpiredJwtException;
+import io.jsonwebtoken.security.SignatureException;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import karuna.karuna_backend.Authentication.JWT.JwtConfig;
+import karuna.karuna_backend.Authentication.JWT.JwtTokenService;
+import karuna.karuna_backend.Errors.DTO.JwtErrorResponse;
+import karuna.karuna_backend.Errors.ErrorKeys.JwtErrorKey;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import java.io.IOException;
+
+@Component
+public class JwtFilter extends OncePerRequestFilter {
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
it can be bean, probably we will use it in diffrent classes

Reply:
I am not sure about it right now to be honest. Let's keep it that way for now at least.